ecs-cli/main.go

```diff
@@ -15,23 +15,45 @@ package main
 
 import (
 	"os"
+	"strings"
 
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/factory"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/cluster"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/compose"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/configure"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/image"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/license"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/log"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/regcreds"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/factory"
+	attributecheckercommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/attributechecker"
+	clusterCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/cluster"
+	composeCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/compose"
+	configureCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/configure"
+	imageCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/image"
+	licenseCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/license"
+	logsCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/log"
+	regcredsCommand "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/regcreds"
```

Comment:
it looks like these import aliases ("regcredsCommand", etc.) are new, but references to them later in the file (e.g., lines 70-82) appear unchanged. Were these updating previously somehow? Wondering how these and their usages could be updated separately.

Reply:
I believe we didn't run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) on this file before.

So I when I made modifications to this file, it automatically restructured our imports :/   
I think it's fine to keep it as is instead of reverting to the previous look since we want to use goimports.